getting-started/kubernetes/vpp/ipsec.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IPsec configuration
+description: Enable IPsec for faster encryption between nodes.
+canonical_url: '/getting-started/kubernetes/vpp/ipsec'
+---
+
+### Big picture
+
+Enable IPsec encryption for the traffic flowing between the nodes.
+
+### Value
+
+IPsec is the fastest option to encrypt the traffic between nodes. It enables blanket application traffic encryption with very little performance impact.
+
+### Before you begin...
+
+In order to enable IPsec encryption, you will need a Kubernetes cluster with:
+- the [VPP dataplane]({{ site.baseurl }}/getting-started/kubernetes/vpp/getting-started) configured
+- [IP-in-IP encapsulation]({{ site.baseurl }}/networking/vxlan-ipip) configured between the nodes
+
+### How to
+
+- [Create the IKEv2 PSK](#create-the-ikev2-psk)
+- [Configure the VPP dataplane](#configure-the-vpp-dataplane)
+
+#### Create the IKEv2 PSK
+
+Create a Kubernetes secret that contains the PSK used for the IKEv2 exchange between the nodes. You can use the following command to create a random PSK. It will generate a unique random key. You may also replace the part after `psk=` with a key of your choice.
```

Comment:
I would have thought user **must** replace with a key..."may" makes it optional.

You may also replace the part after `psk=` with a key of your choice.

Reply:
It is really optional, the default command should generate a safe value.